scripts/update_version.py

```diff
@@ -145,7 +145,6 @@ def main():
     update_files(NODE_ROOT, semver_version)
     update_files(NODE_APP, semver_version)
     update_files(NODE_LIB, semver_version)
-    update_files(NODE_APP_ST_LIB, semver_version)
```

Comment:
Why is this not needed anymore in the `update_version.py` script? Also, it looks like we can remove the `NODE_APP_ST_LIB` variable

Reply:
Yarn workspaces reference each other, so this will now references the library in the workspace https://github.com/streamlit/streamlit/blob/refactor/upgrade-yarn/frontend/app/package.json#L30
which does not change.